crates/sui-core/src/consensus_adapter.rs

```diff
@@ -351,6 +358,9 @@ impl ConsensusAdapter {
                 metrics
                     .sequencing_certificate_control_delay
                     .set(new_delay as i64);
+                metrics
+                    .sequencing_certificate_latency
```

Comment:
Isn't past just the amount of time to send a transaction to consensus?

Reply:
(replied to Lu below)